vm-setup/roles/v1aX_integration_test/files/fetch_manifests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+mkdir -p /tmp/manifests
+
+manifests=(
+  bmh
+  cluster
+  deployment
+  machine
+  machinedeployment
+  machinehealthchecks
+  machinesets
+  machinepools
+  m3clustera
+  m3machine
+  metal3machinetemplate
+  kcp
+  kubeadmconfig
+  kubeadmconfigtemplates
+  kubeadmcontrolplane
+  replicaset
+)
+
+if [[ "${CAPI_VERSION}" == "v1alpha4" ]]; then 
+   manifests+=("m3ippool" "m3ipclaim" "ipaddress" "m3data" "m3dataclaim" "m3datatemplate") 
+fi
+
+for kind in "${manifests[@]}"; do
+  mkdir -p /tmp/manifests/"$kind"
+  for name in $(kubectl get -n metal3 -o name "$kind")
+  do
+    kubectl get -n metal3 -o yaml "$name" > /tmp/manifests/"$kind"/"$(basename "$name")".yaml
```

Comment:
you should maybe add a || true to prevent failing to fetch other objects when one fails

Reply:
Sure, done.